pkg/discovery/servicehandler.go

```diff
@@ -513,16 +514,13 @@ func setOAS3policies(spec *openapi3.T, authPolicy string, configuration map[stri
 }
 
 // isPublished checks if an api is published with the latest changes. Returns true if it is, and false if it is not.
-func isPublished(api *anypoint.API, authPolicy string) (bool, string) {
+func isPublished(api *anypoint.API, authPolicy string, c cache.Cache) (bool, string) {
 	// Change detection (asset + policies)
-	checksum := checksum(api, authPolicy)
```

Comment:
The old way checked if an API Service was published. This worked because we published each instance from mulesoft as a new API Service. Now it does not since we publish the Instances as revisions, so we need to cache it ourselves and check if it is published.